src/NosCore.GameObject/Providers/MapInstanceProvider/MapInstanceProvider.cs

```diff
@@ -99,13 +100,11 @@ public void Initialize()
                         mapinstance.LoadNpcs(npcs[map.MapId]);
                     }
 
-                    mapinstance.StartLife();
                     return mapinstance;
                 }));
 
-            var mapInstancePartitioner =
-                Partitioner.Create(MapInstances.Values, EnumerablePartitionerOptions.NoBuffering);
-            Parallel.ForEach(mapInstancePartitioner, mapInstance => { LoadPortals(mapInstance, portals); });
+            await Task.WhenAll(MapInstances.Values.Select(s=>s.StartLifeAsync())).ConfigureAwait(false);
+            await Task.WhenAll(MapInstances.Values.Select(mapInstance => portals.ContainsKey(mapInstance.Map.MapId) ? LoadPortalsAsync(mapInstance, portals[mapInstance.Map.MapId]) : Task.CompletedTask)).ConfigureAwait(false);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Split this 225 characters long line (which is greater than 200 authorized).](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5326532)